docs/README.md

```diff
@@ -0,0 +1,21 @@
+# The rust-verificatio-tools blog
+
+## Before pushing a new post
+
+[Install jekyll](https://jekyllrb.com/docs/):
+
+``` shell
+sudo apt install ruby ruby-dev
+gem install jekyll bundler --user-install
+bundle config set --local path ~/.local/share/gem/
+bundle install
+```
+
+Check that the blog builds and looks ok:
+
+``` shell
+# cd docs
+bundle exec jekyll serve
```

Comment:
I tend to use --livereload --incremental (or -L -I for short) to:
- only rebuild pages that changed (faster)
- update the web browser automatically on every change (no need to ctrl-R to reload)

Reply:
Also --open-url (-o for short) will automatically open the browser for you.
So the complete invocation is 

> bundle exec jekyll serve -o -I -L

Reply:
(There is no -L option, I assume you meant -l)
Done.